packs/feats/history-of-violence.json

```diff
@@ -32,11 +32,45 @@
             "remaster": true,
             "title": "Pathfinder War of Immortals"
         },
-        "rules": [],
+        "rules": [
+            {
+                "key": "ActiveEffectLike",
+                "mode": "override",
+                "path": "flags.pf2e.historyOfViolence",
+                "value": "floor(@actor.system.abilities.{actor|class.system.keyAbility.selected}.mod)"
```

Comment:
Why the `floor()`? This is just to store the key attribute modifier, right? Maybe we could get that added as a getter since it comes up from time to time instead of building out something like this.

Reply:
> Why the `floor()`? This is just to store the key attribute modifier, right? Maybe we could get that added as a getter since it comes up from time to time instead of building out something like this.

Because partial boosts are stored as a `.5`. I haven't seen key ability *modifier* come up before, but a getter wouldn't be unwelcome!

Reply:
I guess `@Damage` just rounds down the number anyway, so no need for the floor, but I'll just postpone the whole thing until there's a better way to do it.